staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go

```diff
@@ -41,14 +50,40 @@ var (
 
 // QuotaOpts holds the command-line options for 'create quota' sub command
 type QuotaOpts struct {
-	CreateSubcommandOptions *CreateSubcommandOptions
+	// PrintFlags holds options necessary for obtaining a printer
+	PrintFlags *genericclioptions.PrintFlags
+	PrintObj   func(obj runtime.Object) error
+	// The name of a quota object.
+	Name string
+	// The hard resource limit string before parsing.
+	Hard string
+	// The scopes of a quota object before parsing.
+	Scopes           string
+	CreateAnnotation bool
+	FieldManager     string
+	Namespace        string
+	EnforceNamespace bool
+
+	Client         *coreclient.CoreV1Client
+	DryRunStrategy cmdutil.DryRunStrategy
+	DryRunVerifier *resourcecli.DryRunVerifier
+
+	genericclioptions.IOStreams
+}
+
+// NewQuotaOpts creates a new *QuotaOpts with sane defaults
+func NewQuotaOpts(ioStreams genericclioptions.IOStreams) *QuotaOpts {
+	return &QuotaOpts{
+		Hard:       "",
+		Scopes:     "",
```

Comment:
Nit: empty string is the default value for a string :wink:

Reply:
Will do!